codebuild/bin/grep_simple_mistakes.sh

```diff
@@ -43,6 +43,7 @@ KNOWN_MISSES=(
     "s2n_config_set_ctx"
     "s2n_client_hello_has_extension"
     "s2n_async_pkey_op_perform"
+    "s2n_config_get_client_auth_type"
```

Comment:
My test makes use of `s2n_config_get_client_auth_type` which causes grep_simple_mistakes to complain about missing documentation. I figured this exception was ok since I don't expect us to expose `s2n_config_get_client_auth_type()` in the Rust bindings anytime soon. But let me know if you think this is strange and I can think of a different test.